src/System.Linq.Expressions/src/System/Runtime/CompilerServices/CallSiteHelpers.cs

```diff
@@ -12,7 +12,7 @@ namespace System.Runtime.CompilerServices
     /// </summary>
     public static class CallSiteHelpers
     {
-        private static Type s_knownNonDynamicMethodType = typeof(object).GetMethod("ToString").GetType();
+        private static readonly Type s_knownNonDynamicMethodType = typeof(object).GetMethod(nameof(ToString)).GetType();
```

Comment:
I could move it. My thinking was that it's only used once, so we don't need the null checking logic. In fact, it seems like we can use any existing piece of reflection from our cache to do this here; e.g. we already have the `MethodInfo` for `object.GetType` there.